README.md

```diff
@@ -121,13 +121,13 @@ To uninstall the EOSIO built/installed binaries and dependencies, run:
 ```
 
 ## Documentation
-1. [Nodeos](http://eosio.github.io/eos/latest/nodeos/)
-    - [Usage](http://eosio.github.io/eos/latest/nodeos/usage/index)
-    - [Replays](http://eosio.github.io/eos/latest/nodeos/replays/index)
-    - [Chain API Reference](http://eosio.github.io/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index)
-    - [Troubleshooting](http://eosio.github.io/eos/latest/nodeos/troubleshooting/index)
-1. [Cleos](http://eosio.github.io/eos/latest/cleos/)
-1. [Keosd](http://eosio.github.io/eos/latest/keosd/)
+1. [Nodeos](https://developers.eos.io/manuals/eos/latest/nodeos/index)
+    - [Usage](https://developers.eos.io/manuals/eos/latest/nodeos/usage/index)
+    - [Replays](https://developers.eos.io/manuals/eos/latest/nodeos/replays/index)
+    - [Chain API Reference](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index)
+    - [Troubleshooting](https://developers.eos.io/manuals/eos/latest/nodeos/troubleshooting/index)
+1. [Cleos](https://developers.eos.io/manuals/eos/latest/cleos/index)
+1. [Keosd](https://developers.eos.io/manuals/eos/latest/keosd/index)
 
```

Comment:
Thanks @hajunho. @lparisc can we get these fixed please.

Reply:
@heifner I had fixed these broken links on `develop` and `release/2.2.x` branches. I think @hajunho intended a hotfix on `master` as well - although his branch was not based off `master`, hence the other commits. I will cherry-pick his commit to an internal branch then fix `release/2.1.x` and (hotfix) `master` - I haven't had much success with external PRs lately as far passing all buildkite tests.